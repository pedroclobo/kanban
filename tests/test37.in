t 12 análise de requisitos
t 8 desenho
t 34 realização
t 24 teste
t 17 instalação
u pedro
n 1
m 1 pedro IN PROGRESS
n 1
m 2 pedro IN PROGRESS
n 1
m 3 pedro IN PROGRESS
n 1
m 4 pedro IN PROGRESS
n 1
m 5 pedro IN PROGRESS
l
n 1
m 1 pedro DONE
n 1
m 2 pedro DONE
n 1
m 3 pedro DONE
n 1
m 4 pedro DONE
n 1
m 5 pedro DONE
l
q
