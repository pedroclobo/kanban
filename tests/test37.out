task 1
task 2
task 3
task 4
task 5
1
2
3
4
5
1 IN PROGRESS #12 análise de requisitos
2 IN PROGRESS #8 desenho
5 IN PROGRESS #17 instalação
3 IN PROGRESS #34 realização
4 IN PROGRESS #24 teste
6
duration=5 slack=-7
7
duration=5 slack=-3
8
duration=5 slack=-29
9
duration=5 slack=-19
10
duration=5 slack=-12
1 DONE #12 análise de requisitos
2 DONE #8 desenho
5 DONE #17 instalação
3 DONE #34 realização
4 DONE #24 teste
